src/schemas/schemas.ts

```diff
@@ -4,14 +4,42 @@ import { StatusCodes } from 'http-status-codes';
 // local
 import { MIN_ACTIONS_SAMPLE_SIZE, MAX_ACTIONS_SAMPLE_SIZE } from '../constants/constants';
 
+// todo: get from graasp-utils
 export const uuid = S.string().pattern(
```

Comment:
Is this in the SDK?

Reply:
I still don't know how to manage the schemas, so I didn't touch it even in the sdk refactor. We can discuss.